txpipe/twopoint_fourier.py

```diff
@@ -221,11 +223,85 @@ def load_maps(self):
         syst_nc = None
         syst_wl = None
 
+        
+        # Load HEALPix systematics maps
+        syst_nc = self.config['syst_nc']
+        syst_wl = self.config['syst_wl']
+        if syst_nc:  
+            print('Deprojecting systematics maps for number counts')
+            n_systmaps = 0
+            s_maps = []
+            systmaps_dir = self.config['systmaps_nc_dir']
+            systmaps_path = pathlib.Path(systmaps_dir)
+            try:
+                for systmap in systmaps_path.iterdir():
+                    if systmap.is_file():
+                        systmap_file = str(systmap)
```

Comment:
maybe also check if the filename ends with .fits or .hs here and only use it if so?

Reply:
I'll check that it is a .fits file since at present only HEALPix is implemented (we can add healsparse later when TXPipe supports it).